learn/getting_started/quick_start.md

```diff
@@ -250,180 +261,11 @@ If you have multiple indexes, you can switch between them using the indexes drop
-
-To use `instant-meilisearch` in `Vue.js` using `npm` or `yarn` please visit [meilisearch-vue](https://github.com/meilisearch/meilisearch-vue).
-
-:::
-
-::: tab React
-
-The following code sample uses [React](https://reactjs.org/) framework.
-
-```html
-<!DOCTYPE html>
-<html lang="en">
-  <head>
-    <meta charset="utf-8" />
-    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@meilisearch/instant-meilisearch/templates/basic_search.css" />
-  </head>
-  <body>
-      <div id="app" class="wrapper"></div>
-  </body>
-  <script src="https://unpkg.com/react@16/umd/react.development.js" crossorigin></script>
-  <script src="https://unpkg.com/react-dom@16/umd/react-dom.development.js" crossorigin></script>
-  <script src="https://cdn.jsdelivr.net/npm/react-instantsearch-dom@6.7.0/dist/umd/ReactInstantSearchDOM.js"></script>
-  <script src="https://cdn.jsdelivr.net/npm/@meilisearch/instant-meilisearch/dist/instant-meilisearch.umd.min.js"></script>
-  <script>
-    const { InstantSearch, SearchBox, Hits, Highlight, Configure }  = ReactInstantSearchDOM;
-    const searchClient = instantMeilisearch(
-      "http://localhost:7700"
-    );
-
-    const App = () => (
-      React.createElement(InstantSearch, {
-        indexName: "movies",
-        searchClient: searchClient
-      }, [
-        React.createElement(SearchBox, { key: 1 }),
-        React.createElement(Hits, { hitComponent: Hit, key: 2 }),
-        React.createElement(Configure, { hitsPerPage: 10 })]
-      )
-    );
-    function Hit(props) {
-        return React.createElement(Highlight, {
-          attribute: "title",
-          hit: props.hit
-        })
-    }
-    const domContainer = document.querySelector('#app');
-    ReactDOM.render(React.createElement(App), domContainer);
-  </script>
-</html>
-```
-
-The code above comes in multiple parts:
-
-- The `<body>` of the page is the entry point for React. `instant-meilisearch` adds the search bar and search results here by manipulating the DOM.
-- `<script src="..">` tags are [CDNs](https://en.wikipedia.org/wiki/Content_delivery_network) that import libraries needed to run `instant-meilisearch` in [React](https://reactjs.org/).
-- The `<script>` containing JavaScript initialize React and renders the code that will be rendered in the body. Customization of `instant-meilisearch` happens here as well.
-
-To use `instant-meilisearch` in `React` using `npm` or `yarn` please visit [meilisearch-react](https://github.com/meilisearch/meilisearch-react).
-
-:::
-
-::::
-
 ### Let's try it!
 
 1. Create an `html` file, for example, `index.html`
 2. Open it in a text editor (e.g. Notepad, Sublime Text, Visual Studio Code)
-3. Copy-paste any of the code examples below and save the file
+3. Copy-paste any of the code examples above and save the file
 4. Open `index.html` in your browser (double click on it in your folder)
 
 You should now have a working front-end search interface 🚀🔥
```

Comment:
We should delete these lines as well. We can replace them with a line like this:

```
To learn more about instant-meilisearch, refer to its [documentation](https://github.com/meilisearch/instant-meilisearch#readme) located in the project repository.
```